### ISSUE TITLE
Limited multiplier

### ISSUE THREAD

**Issue by Iankid** [ironmonk88/TerrainLayer#6]
Currently, the module only allows modifiers of 0.5 to 4.
This is an issue as, I was hoping to make use of the Difficult Terrain module's ability to be additive from this mod, rather then mutilative. At least until that other issue with the line multiplying the entire travel path is fixed.
But even if that other issue is fixed, the option to select the range of the multiplier selection could be of potential use. Perhaps being added to the Mod's settings menu.

**Comment by Ikabodo** [comment]
I was just wondering the same thing.
The ability to manually configure the "multipliers" to for example 5, 10, 15 seems like a really neat feature that would work well with dragging.

And it would also solve the long running problem of a terrain-box multiplying the entire ruler.

**Comment by ironmonk88** [comment]
Problem with this is I'm trying to keep static values out of the Terrain Layer and leave it as relative values.  
It's up to the Ruler to determine cost and to keep track of what measurement is being used within the system.
Terrain Layer just keeps track of the cost vs. what a regular cost would be.

I was also keeping it limited due to the use of specific images for each difficulty, though I suspect I could just cap it at the smallest/largest ones and you'd have to show numbers if you changed it to anything different.

**Comment by Ikabodo** [comment]
Ok! Sorry if I'm not understanding what module does what here. :) Maybe this feedback belongs to the drag-ruler module?

About the specific images, that makes sense too. I guess I would solve it with having a certain "fixed" number of levels. It's just up to the options to decide what each level does. I.e. 1,2,4,8 or 5,10,15 and so on.

Anyway, the core issue I'm having is not the lack of additive, it's that the difficult terrain is multiplied on the whole ruler. Which more or less removes the use for the difficult terrain layers for my players. (they are not savvy enough to be able to figure out that they have to "stop" and "start" and so on. But I think this feedback either belongs with the other modules, or as a new one?

I guess other DMs just use difficult terrain as one big multiplyer? as in, if you pass through a difficult terrain box at least once, half your speed.

Anyway, thanks for considering it ironmonk!

**Comment by Ikabodo** [comment]
In my mind, this issue is no longer valid. Since [the latest DragRuler](https://github.com/manuelVo/foundryvtt-drag-ruler/releases/tag/v1.4.0) solved the issue with the whole ruler multiplying. We no longer need the difficult terrain module.

**Comment by ironmonk88** [comment]
Yeah, I was thinking it was really strange how Difficult Terrain handled things.  I can barely remember to set waypoints, so I'm sure my players would have issues.

I think the combination of DragRuler, TerrainRuler, and TerrainLayer should have most things covered.

And I should really get my version of Terrain Layer published soon.
